app/store/sagas/login.js

```diff
@@ -58,7 +56,7 @@ export function* authenticationFlowSaga() {
       const navigation = yield take('Navigation/NAVIGATE');
       if (navigation.routeName === 'Home' && navigation.params.code !== undefined) {
         let token = yield call(authorize, navigation.params.code);
-        if (token) {
+        if (token !== undefined) {
```

Comment:
it can't if follow the partern we talked about previously 
const {error, token}=call(.....)